Makefile

```diff
@@ -220,4 +220,3 @@ code-generate:
 	./hack/generate.sh
 
 include Makefile.docker
-include Makefile.codegen
```

Comment:
So we do still need/want codegen for the deepcopy structs for `pkg/tekton/syntax`, `pkg/jenkinsfile`, and `pkg/config` (for `ProjectConfig` in the last case).